src/components/alert/alert.style.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import StyledIconButton from '../icon-button/icon-button.style';
+
+const AlertStyle = styled.div`
+  ${StyledIconButton} {
+    position: absolute;
+    top: 25px !important;
```

Comment:
Can we use `&&` instead of `!important` here?

Reply:
fixed: 387f75c